db/__tests__/seed.test.js

```diff
@@ -333,4 +321,31 @@ test('DELETE 400: responds with error message when comment_id is invalid', () =>
     .delete('/api/comments/fdsa')
     .expect(400)
 })
+})
+describe.only('/api/users', () => {
+test('GET 200: responds with array of users', () => {
+    return request(app)
+    .get('/api/users')
+    .expect(200)
+    .then(({ body }) => {
+    const { users } = body
```

Comment:
We need a length check here before the forEach, without this we could be getting an empty array back, meaning the forEach never runs and so the expect never runs and we get a false positive.